doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -235,6 +235,10 @@ Modem libraries
 
   * Added an optional ``family`` parameter to :c:func:`pdn_activate`, which is used to report when the IP family of a PDN changes after activation.
 
+* A-GPS library:
+
+  * The A-GPS library has been deprecated in favor of using the assistance libraries directly.
```

Comment:
You could add link to the one that should be used instead of the deprecated library.

Reply:
Added.